packages/@react-native-windows/cli/tsconfig.json

```diff
@@ -8,10 +8,14 @@
       "DOM.Iterable",
       "ES2019.String"
     ],
-
     // Not clean
     "strictFunctionTypes": false,
   },
-  "include": ["src"],
-  "exclude": ["node_modules"]
-}
+  "include": [
+    "src",
+    "../telemetry/src/utils/nameHelpers.ts"
```

Comment:
You don't want to cross include packages, like this, don't include this here. You won't need it if you properly export the nameHelpers functions from the telemetry package, and remember to re-build the telemetry package so you can see them in this package.